replica/table.cc

```diff
@@ -2292,6 +2292,65 @@ std::vector<sstables::shared_sstable> table::select_sstables(const dht::partitio
     return _sstables->select(range);
 }
 
+future<> table::drop_quarantined_sstables()
+{
```

Comment:
nit: out coding style calls for putting the curly brace on the same line as the function